10_date_timestamps_and_lies.md

```diff
@@ -106,8 +106,6 @@ val df = sqlContext.read.format("jdbc")
   .load
 ```
 
-But ***why doesn't casting work ?***
```

Comment:
There is nothing about casting. IMHO Instead of explaining this issue here it would be better to put in next to predicate pushdown.